### PR TITLE
Fix grade command User Guide

### LINKED ISSUE #214: [PE-D][Tester B] UG - Unexpected behaviour for grade command
UG states that for `grade` command, the parameter `g/SCORE` need to be below the maximum marks for that particular assignment and above 0.

**Steps taken:**
1. Grade student with `grade 2 n/assignment1 g/0`

**Expected**: Assignment score should be between 1 to max score.

**Actual**: Assignment graded successfully!

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: saltedfishxx/ped#8

### PR DESCRIPTION
**Pull request by Vanessamae23**
## Description

Fix grade command User Guide Closes #214 

## Context

Fix grade command User Guide to include 0 and max score boundary (inclusive)

## Checklist

- [ ] I have self-reviewed my changes.
- [ ] I have added JavaDocs to all relevant methods.
- [x] I have updated the documentation: 
  - User Guide Table of Contents, Description and Summary.
  - Developer Guide, if applicable.
- [ ] I have ran `./gradlew test` or `./gradlew check` and all checks pass.
- [ ] I have updated my project portfolio with what I have contributed.

### CODE REVIEW

**Review by et-irl — APPROVED** [review]
Yes, the intended behaviour was to allow 0, and the user guide was wrong. Good fix!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -613,7 +613,7 @@ Grade a student's assignment.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Extra Information**
 <br><br>
-- A student's marks need to be below the maximum marks for that particular assignment and above 0.
+- A student's marks need to between 0 (inclusive) and the maximum score assigned to the assignment (inclusive).
```
Comment (et-irl) [inline docs/UserGuide.md:616]: - A student's marks need to between 0 (inclusive) and the maximum score **_of_** the assignment (inclusive).

Would the above sound better? It sounds a bit terse.

**Review by et-irl — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (proto-aiken-13) [reply docs/UserGuide.md]: Here would be the corrected grammar (as well as a suggested line):
-  Inputted Marks must **be** at least 0 and not exceed the maximum assignment score.

Would it sound better? Just suggesting.

**Review by proto-aiken-13 — APPROVED** [review]
lgtm

**Review by et-irl — APPROVED** [review]
great!

**Pull request merged**